SetupConan.cmake

```diff
@@ -13,20 +13,34 @@ list(GET VERSION_LIST 2 COMPILER_VERSION_PATCH)
 
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
     message(STATUS "Setup dependencies with conan (clang ${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR})")
-    set(CONAN_COMPILER "clang")
+    set(INSTALL_ARGS
```

Comment:
@Morheit , please change this block the code below. Your variant doesn't work. Pay attention that in my veriant quotes are missing (this is important). Also I've removed some non-mandatory settings from cmd

```
if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
    message(STATUS "Setup dependencies with conan (clang ${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR})")
    set(INSTALL_ARGS
        -s compiler=clang
        -s compiler.version=${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR}
        -s compiler.libcxx=libstdc++11
        -s build_type=Release
        -e CXX=${CMAKE_CXX_COMPILER}
        -e CC=${CMAKE_C_COMPILER}
        --build missing)
elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
    message(STATUS "Setup dependencies with conan (gcc ${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR})")
    set(INSTALL_ARGS
        -s compiler=gcc
        -s compiler.version=${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR}
        -s compiler.libcxx=libstdc++11
        -s build_type=Release
        -e CXX=${CMAKE_CXX_COMPILER}
        -e CC=${CMAKE_C_COMPILER}
        --build missing)
```

Also I propose you to test new version under linux, e.g. in docker. you can use docker file available in this project.

Reply:
@malirod, I've tested and pushed your proposed solution. Looks nice and seems to be workable.